src/js/media-container.js

```diff
@@ -25,6 +25,7 @@ template.innerHTML = `
 
       /* Default dimensions */
       width: 720px;
+      max-width: 100%;
```

Comment:
What does this fix?

Reply:
This makes for a better default on smaller screens (< 720px width, like an iphone 6 that is 375px), so that it doesn't overflow horizontally

Reply:
Oh cool, that's great. For things like that I like to add a quick comment note, otherwise the next person optimizing might strip it out not knowing why it's there.

Reply:
updated 😄